containers/gateway/content/run.sh

```diff
@@ -25,5 +25,15 @@ then
   . ~/startup.sh
 fi
 
-# Start the DataLab kernel gateway
-jupyter kernelgateway --JupyterWebsocketPersonality.list_kernels=True --KernelGatewayApp.port=8080 --KernelGatewayApp.ip=0.0.0.0
+# Start the DataLab kernel gateway.
+jupyter kernelgateway --JupyterWebsocketPersonality.list_kernels=True --KernelGatewayApp.port=8081 --KernelGatewayApp.ip=0.0.0.0 &
```

Comment:
For now I added some script to make sure Jupyter is up and running within 10 seconds.